### ISSUE TITLE
Static interface members + assembly view

### ISSUE THREAD

**Issue by controlflow** [ashmind/SharpLab#748]
https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzmAPgAQAwAIDKAFhAE4AOAMhMAHQBKArjAlGAKYDcAsAFC+wI2JAGYQAxm1wBJXAG9euXJgDMuaigQlxCJQEYAbLgYDcAMQAUASm48Avr16YATLgDCuENLkKlqzAaMTC0tcAF4APlxsAA8AEQBRAEFYgCF4+NMbez4eAG0AKSgEAHE2GCEoMXMEAE8yNgB7YXNXS0sAXUcXYoAeABVIgHdCIUk+jy95HkUVPUNjZjMrMMi+mmCsoA=

```c#
using SharpLab.Runtime;

interface I {
  public abstract static uint F();
}

class C : I {
  public static uint F() => 0xDEADBEEF;
}

[JitGeneric(typeof(C))]
class G<T> where T : I {
  public static uint F() => T.F();
}
```

Assembly view results in error:
```
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Virtual Static Method.
Virtual Static Method.
Virtual Static Method.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.RuntimeAssembly.get_DefinedTypes()
   at SharpLab.Server.Decompilation.JitAsmDecompiler.ValidateStaticConstructors(Assembly assembly) in D:\a\SharpLab\SharpLab\!roslyn-branches\dotnet-main\sharplab\source\Server\Decompilation\JitAsmDecompiler.cs:line 57
   at SharpLab.Server.Decompilation.JitAsmDecompiler.Decompile(CompilationStreamPair streams, TextWriter codeWriter) in D:\a\SharpLab\SharpLab\!roslyn-branches\dotnet-main\sharplab\source\Server\Decompilation\JitAsmDecompiler.cs:line 38
   at SharpLab.Server.MirrorSharp.SlowUpdate.WriteResult(IFastJsonWriter writer, Object result, IWorkSession session) in D:\a\SharpLab\SharpLab\!roslyn-branches\dotnet-main\sharplab\source\Server\MirrorSharp\SlowUpdate.cs:line 177
   at MirrorSharp.Internal.Handlers.SlowUpdateHandler.SendSlowUpdateAsync(IReadOnlyList`1 diagnostics, WorkSession session, Object extensionResult, ICommandResultSender sender, CancellationToken cancellationToken)
   at MirrorSharp.Internal.Handlers.SlowUpdateHandler.ExecuteAsync(AsyncData data, WorkSession session, ICommandResultSender sender, CancellationToken cancellationToken)
   at MirrorSharp.Internal.Connection.ReceiveAndProcessInternalAsync(CancellationToken cancellationToken)
   at MirrorSharp.Internal.Connection.ReceiveAndProcessAsync(CancellationToken cancellationToken)
```

**Comment by svick** [comment]
I think this is a consequence of attempting to load code that uses a .Net 6 feature on .Net 5, [which is what SharpLab uses at the moment](https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK4wD4AEBMBGAWAChis8BOACjIDoAldBKMAUxoEkYEXkB7ABwDKPAG5QAxiwDO9RsxacAZr2SQmvGDQBiyCKwDuKgNYARaeORR+6mAEoA3EA==).

**Comment by ashmind** [comment]
Thanks for raising this! As @svick mentioned, this is related to SharpLab using .NET 5.
It would probably make sense to run branch sites under absolute latest .NET, but there is some complexity in doing that, so for now I just plan to update to .NET 6 once it is released and available on Azure.

**Comment by ashmind** [comment]
This is now resolved by #879 (migration to .NET 6).
